exercises/pov/description.md

```diff
@@ -43,4 +43,4 @@ This exercise involves taking an input tree and re-orientating it from the point
 of view of one of the nodes.
 
 [wiki-graph]: https://en.wikipedia.org/wiki/Tree_(graph_theory)
-[wiki-tree]: https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)
\ No newline at end of file
+[wiki-tree]: https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)
```

Comment:
```diff
diff --git a/exercises/pov/description.md b/exercises/pov/description.md
index 74e2e5c..cfdd2ae 100644
--- a/exercises/pov/description.md
+++ b/exercises/pov/description.md
@@ -43,4 +43,4 @@ This exercise involves taking an input tree and re-orientating it from the point
 of view of one of the nodes.
 
 [wiki-graph]: https://en.wikipedia.org/wiki/Tree_(graph_theory)
-[wiki-tree]: https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)
\ No newline at end of file
```

Reply:
I'm not sure what you mean here - the file currently has this reference URL on both main and this development branch, and the reference is used in the text (line # 5).

Reply:
I did not expect the file in two places in the patch, and really should have double checked.  If possible, the 12 commits could be rebased and organized and some of them can be squashed to make the story flow.  But, if this is merged and squashed, then the story can be told at that point as intended.

Sorry for the additional noise.

Reply:
Yep, this PR should definitely be squashed when merging

No worries, I learned a new feature of GitHub!